xla/service/gpu/fusions/transforms/BUILD

```diff
@@ -48,7 +49,12 @@ cc_library(
         "simplify_arith.cc",
         "unswitch_loops.cc",
         "vectorize_loads_stores.cc",
-    ],
+    ] + if_cuda_is_configured(
+        [
+            "convert_float_nvidia.cc",
+        ],
+        no_cuda = ["cuda_stub.cc"],
+    ),
```

Comment:
Since the .cc file now does not depend on CUDA-only targets anymore, it can be a normal src file, right?

Also the stub is not needed anymore, right?

Reply:
Indeed, that was the whole point. Done.